### PR TITLE
SetHostnameAsFQDN option is added to all components. Default is true

### PR DESCRIPTION
**Pull request by qurname2**

### CODE REVIEW

**Comment by kruftik** [comment]
it seems the added options isn't used in podtemplate generation. it still has the hardcoded true value there

**Review by l0kix2 — CHANGES_REQUESTED** [review]
Agree with @kruftik, it looks that it must be passed deeper.
And also, maybe we can set this default in one place in the server, not every component?
Or use default in spec

**Comment by l0kix2** [comment]
I suggest we use spec default true for simplicity and check for the diff when will update ytop

File: pkg/apiproxy/ytsaurus.go [hunk 1]
```diff
@@ -140,10 +140,10 @@ func (c *Ytsaurus) IsStatusConditionFalse(conditionType string) bool {
...
-	    statusOrder := []metav1.ConditionStatus{metav1.ConditionTrue, metav1.ConditionFalse, metav1.ConditionUnknown}
-	    if diff := cmp.Compare(slices.Index(statusOrder, a.Status), slices.Index(statusOrder, b.Status)); diff != 0 {
-		return diff
-	    }
-	    return a.LastTransitionTime.Compare(b.LastTransitionTime.Time)
+		statusOrder := []metav1.ConditionStatus{metav1.ConditionTrue, metav1.ConditionFalse, metav1.ConditionUnknown}
+		if diff := cmp.Compare(slices.Index(statusOrder, a.Status), slices.Index(statusOrder, b.Status)); diff != 0 {
+			return diff
+		}
+		return a.LastTransitionTime.Compare(b.LastTransitionTime.Time)
```
[the remark this answers is not in the document]
Comment (qurname2) [reply pkg/apiproxy/ytsaurus.go:147]: Looks like not

**Comment by qurname2** [comment]
> Agree with @kruftik, it looks that it must be passed deeper. And also, maybe we can set this default in one place in the server, not every component? Or use default in spec
@l0kix2 yes, I think we can set true in server and check InstanceSpec if this option is exists

**Review by l0kix2 — APPROVED** [review]
LGTM if default works and we not making field = false for users.

**Comment by l0kix2** [comment]
Fix #295

**Pull request merged**